WcaOnRails/spec/models/user_spec.rb

```diff
@@ -109,8 +109,10 @@
   end
 
   it "allows senior delegate if board member" do
-    board_member = FactoryBot.create :user
-    board_member.board_member!
+    board_team = Team.board
+    board_member = FactoryBot.create :delegate
```

Comment:
Can you make `board_member` a trait on the user factory instead? That way, you should be able to combine this line with the line below and just do:

```ruby
board_member = FactoryBot.create :delegate, :board_member
```

I think in general, we prefer using traits to factories, because you can compose them like this.